ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -26,6 +26,12 @@ then
 	batt_smbus start -X
 fi
 
+# Start batmon driver if enabled using BATMON_DRIVER_EN
+if param compare -s BATMON_DRIVER_EN 1
+then
+	batmon start -X -b 4  #start on external bus
```

Comment:
I don't think this should specify the bus number, doesn't that depend on the board?

Reply:
Most people use batmon with pixhawk 4 board on the external bus 4. We want it to work for this configuration with minimal overhead. 

For some reason we weren't able to get the driver to pickup the default external bus without explicitly specifying it. Will fix this on a later PR.

Reply:
Dev call:
batmon `start -X` should look on all I2C buses and find the device. But reportedly batmon does not connect if the `-b` bus is not specified. That has to be checked.

Reply:
resolved